src/alttxt/tokenmap.py

```diff
@@ -396,15 +396,19 @@ def max_n_intersections(self, n: int) -> str:
                 break
             
             formatted_names = self.truncate_separately(sort[i].name)
-            result += f"{formatted_names} ({sort[i].size}), "
-        
-            if i == n - 2:
-                result += "and "
+            result += f"{formatted_names} ({sort[i].size}); "
+
+        result = result.rstrip("; ")
+        parts = result.split("; ")
+
+        if len(parts) > 2:
+            result = "; ".join(parts[:-1]) + "; and " + parts[-1]
```

Comment:
Why use `;` instead of `,`?

Reply:
https://github.com/visdesignlab/upset-alt-txt-gen/issues/65

that is what the plan for fix of the issue is

Reply:
Thanks for clarifying. I'd suggest filling that information in on the PR description above. Don't need to do it retroactively for this PR though